src/paths/projects.ts

```diff
@@ -22,12 +24,23 @@ ProjectsRouter.get(
     })
 );
 
-ProjectsRouter.use('/:id', prefetch(Project));
-
-ProjectsRouter.get('/:id', read(Project));
-
-ProjectsRouter.get('/:id/metadata', (req, res) => {
-    const project = req.doc as HydratedDocument<IProject>;
+ProjectsRouter.get('/:tokenId/metadata', async (req, res) => {
+    const tokenId = req.params.tokenId;
+    if (tokenId === undefined) {
+        return res.status(404).end();
+    }
+    const provider = ethers.getDefaultProvider(chainId);
+    const contract = new ethers.Contract(contractAddress, abi, provider);
+    const [_status, _fee, _balance, _owner, id] =
```

Comment:
I would have this as, to conform to the eslint config. Added a comment to keep track of others for future reference
```suggestion
// Returned array is of following form[status, fee, balance, owner, id]
const [, , , , id] = (await contract.editions(tokenId)) ?? [];
```

Reply:
Ah sorry I didnt see these comments